classes/hooks/synchronization/Initial.php

```diff
@@ -83,14 +83,14 @@ public function synchronize()
             array('Datatype' => 'str', 'Name' => $segmentationObject->ll(49), 'NameSpace' => 'static')
         ));
 
-        $file = new SplFileObject('contacts.csv', 'w');
+        $file = fopen('contacts.csv', 'w');
```

Comment:
Instead of creating a temporary file you should directly store contacts informations in a variable.